package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/ccd-case-ui-toolkit",
-  "version": "3.1.5-complex-field-check-answers",
+  "version": "3.1.5-profile-call-before-event-trigger",
```

Comment:
Is this correct? On main the current version  "3.1.5-complex-field-check-answers"
Should we not be incrementing the version

Reply:
we don't do version increment only postfix ticket reference and the same confirmed with @Mo-Lala84